src/server.c

```diff
@@ -2616,6 +2630,15 @@ void populateCommandStructure(struct redisCommand *c) {
         c->key_specs_num++;
     }
 
+    /* Count things so we don't have to use deferred reply in COMMAND reply. */
+    if (c->history)
+        while (c->history[c->num_history].since)
+            c->num_history++;
```

Comment:
can be done in one line

Reply:
you mean doing the increment inside the `while`?
i thought about it, but then it means it doesn't have a body (just `;`), and i don't like that.